runtests.py

```diff
@@ -56,14 +56,10 @@
     # Self type check
     "self": [
         executable,
-        "-m",
-        "mypy",
-        "--config-file",
-        "mypy_self_check.ini",
-        "-p",
-        "mypy",
-        "-p",
-        "mypyc",
+        "-c",
+        "from mypy.main import main as mypy; "
+        "mypy(args=['--config-file', 'mypy_self_check.ini', '-p', 'mypy', '-p', 'mypyc']); "
+        "mypy(args=['--config-file', 'mypy_self_check.ini', 'setup.py'])",
```

Comment:
maybe we should have two different type checks, `self` (which checks mypy) and `self-packaging` (which checks `setup.py`)?

Reply:
I also thought about splitting them, but then I saw the comment at https://github.com/python/mypy/pull/18609/files/72e1ff81760a6e50a2acba2d2a262ad409afdc25#diff-17019d8813f584a97bc5531faa1df3f1a590413d61a77d7d93db76df66163b63R53-R54

> Each run should have tests that each take a roughly similar time to run.

To me, the check of just `setup.py` feels too short to have its own run.

Reply:
I'm happy to take this PR in whatever direction the reviewers / mergers / maintainers wish, of course :-)

Reply:
Fixed. Shall I squash+rebase and force push?